Cargo.toml

```diff
@@ -45,3 +45,6 @@ exclude = [
     'iml-system-rpm-tests',
     'iml-system-docker-tests',
 ]
+
+[profile.release]
+debug = true
```

Comment:
:-)

Reply:
Yep, this is for profiling to give function names